src/plugins/intel_cpu/thirdparty/llmdnn/tests/src/test_utility_kernel_avx512.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
why do we need to put tests files at all?

Reply:
The library is like a third-party component for openvino. It has its own unit tests, performance tests, and all internal tests also sit inside the directory to ensure the library works well. Maybe it's better to be moved to a separate repo.

Reply:
The library is coming from the [special optimized branch](https://github.com/luo-cheng2021/openvino/tree/luocheng/gpt_neox/src/plugins/intel_cpu/src/nodes/special) for customer, here is the only place to host the source code. It's better to add the tests in our CI, could we do so? @dmitry-gorokhov

Reply:
As we agreed the library will be pushed in separate repository and integrated inside OV as submodule (or via TBB appraoch)
On OV level these MatMul impls should be tested via existing single layer tests approach and enabled in precommit.
I think we can also leave own llmdnn tests within the library itself if they are required for development. Adding them into CI is a separate topic.